react/components/ProductMessagesProgress.tsx

```diff
@@ -147,15 +184,15 @@ const ProductMessagesImport: FC<Props> = ({
         <p>{getCurrentProgressPercentage(progress, total)}</p>
         {progress < total ? (
           <Button variation="secondary" onClick={cancel}>
-            {intl.formatMessage({ id: 'admin/messages.cancel' })}
+            {intl.formatMessage(cancelMessage)}
```

Comment:
non-blocking: all intl are messages, so you don't need to repeat "Message" in the translation key. `cancel` alone would suffice